db/migrate/20220223231753_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :likes do |t|
```

Comment:
- [ ] Let's follow the Ruby naming convention, postId should be in a snail case `post_id`

Reply:
done